apps/backend/src/db/migrations/20240827214519_add_files_table.ts

```diff
@@ -0,0 +1,35 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	FILES: "files",
+} as const;
+
+const ColumnName = {
+	CREATED_AT: "created_at",
+	FILE_KEY: "file_key",
```

Comment:
What is it and why do we need it?

Reply:
What do you mean? Are you about `FILE_KEY`?

We need `FILE_KEY` to store the file's name on the bucket.

Reply:
Yes, but why do we need to keep it on our DB?